integration_test/test_videoroom/test/support/simulcast_mustang.ex

```diff
@@ -0,0 +1,120 @@
+defmodule SimulcastMustang do
+  use Stampede.Mustang
+
+  @impl true
+  def join(browser, options) do
+    page = browser |> Playwright.Browser.new_page()
+    _response = Playwright.Page.goto(page, options.target_url)
+
+    Process.sleep(500)
+
+    :ok = Playwright.Page.click(page, "[id=#{options.start_button}]")
+
+    {browser, page}
+  end
+
+  @impl true
+  def afterJoin({browser, page}, options) do
+    Process.sleep(options.join_interval)
+
+    buttons = Map.get(options, :start_buttons, [])
+
+    for button <- buttons do
+      :ok = Playwright.Page.click(page, "[id=#{button}]")
+    end
+
+    {browser, page}
+  end
+
+  @impl true
+  def linger({_browser, _page} = ctx, options) when options.buttons == [] do
+    Process.sleep(options.linger)
+    ctx
+  end
+
+  @impl true
+  def linger({_browser, page} = ctx, options) do
+    for {{button, timeout}, stage} <- options.buttons do
+      case button do
+        stats when stats in ["simulcast-own-low", "simulcast-own-medium", "simulcast-own-high"] ->
+          get_stats(page, options.receiver, options.id, stage, options.sender_button)
+
+          Process.sleep(1_000)
+
+          :ok = Playwright.Page.click(page, "[id=#{button}]")
+
+          measurments(options.sender_button, page, options, stage, timeout - 2_000)
```

Comment:
Because earlier actions `get_stats` and `Process.sleep` took us 2 seconds to finish. The user of mustang expects that test will take x seconds, so it shouldn't take a lot longer.